pkg/reconciler/trigger/trigger.go

```diff
@@ -28,6 +28,7 @@ import (
 	apierrs "k8s.io/apimachinery/pkg/api/errors"
 
 	"github.com/google/knative-gcp/pkg/logging"
+	reconcilerutils "github.com/google/knative-gcp/pkg/reconciler/utils"
```

Comment:
nits: can we put this together with other `pkg/reconciler/utils` imports?

Reply:
done as suggested